README.md

```diff
@@ -44,6 +44,12 @@ jupyter-lite build --output-dir dist
 
 Analysim requires two databases to operate: one SQL database (PostgreSQL) for relational data and one Azure BlobStorage database for keeping uploaded user files. In addition, an Outlook account is needed for the email functionality. All of these services are accessed via authentication information stored in the `appsettings.json` and `appsettings.Development.json` files under the `src/Analysim.Web` folder. The structure of the files are as follows (`XXX` means redacted):
 
+1. Create `appsettings.json` and `appsettings.Development.json`
+
+- Navigate to `src\Analysim.Web` folder in your cloned repository.
+- Create two files name `appsettings.json` and `appsettings.Development.json` if they do not exist.
+- Paste the following JSON structure into both files.
+
```

Comment:
@AashifAmeer this is just repeating the same information from the paragraph above. And it's also not flowing correctly because the previous paragraph ends in a colon. You may need to merge them better.